sources/packages/web/nginx.conf

```diff
@@ -0,0 +1,30 @@
+user  nginx;
+worker_processes  1;
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+events {
+  worker_connections  1024;
+}
+http {
+  include       /etc/nginx/mime.types;
+  default_type  application/octet-stream;
+  log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                    '$status $body_bytes_sent "$http_referer" '
+                    '"$http_user_agent" "$http_x_forwarded_for"';
+  access_log  /var/log/nginx/access.log  main;
+  sendfile        on;
+  keepalive_timeout  65;
+  server {
+    listen       8080;
+    server_name  localhost;
+    location / {
+      root   /app;
+      index  index.html;
+      try_files $uri $uri/ /index.html;
+    }
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
```

Comment:
Just for my understanding, what is `/50x.html`. is there an HTML file?

Reply:
That is the html file for the 50x errors. We don't have one. Should we create a place holder one or remove that configuration for now? Ideas? @dheepak-aot @guru-aot @andrewsignori-aot  @sh16011993

Reply:
I would be ok to have it removed. We can track improvements in a new ticket to be created.